src/components/containers/ProfileSettings/ProfileSettings.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { Form } from 'semantic-ui-react';
+
+// components
+import Button from '../../presentationals/Button/Button';
+
+class Settings extends Component {
```

Comment:
This should be the same with the name of the file.

Reply:
This should be a stateless component as we are using any local state here.

Reply:
This has been made a stateless component and moved to the presentationals folder